solidity/contracts/libs/isms/MultisigIsmMetadata.sol

```diff
@@ -88,4 +67,65 @@ library MultisigIsmMetadata {
         uint256 _end = _start + SIGNATURE_LENGTH;
         return _metadata[_start:_end];
     }
+
+    function suffix(
```

Comment:
How much less gas efficient is it for this to return the actual suffix as a bytes slice, and then to take slices from that? I think it would be much more readable